### ISSUE TITLE
Consider changes in usedBy

### ISSUE THREAD

**Issue by libremente** [publiccodeyml/publiccode.yml#61]
As of now, it is not clear if the `usedBy` key should contain also the name of the PA which owns the copyright or not. Probably it should be clearer.

**Comment by johanlinaker** [comment]
In addition, I would propose either 
* removing usedBy and enabling a meta-standard which each national catalouge can maintain, or
* having usedBy per nation, e.g., swe/usedBy as with descriptions and features
Let me know if I should create a new issue for this.

**Comment by lsmith77** [comment]
I agree with removing, see also https://github.com/publiccodeyml/publiccode.yml/pull/187

**Comment by bfabio** [comment]
Proposal in #187.
